File: src/main/java/no/jstien/bikeapi/utils/VarUtils.java
package no.jstien.bikeapi.utils;

public class VarUtils {
    public static String getEnv(String key) {
        String tmp = null;
        tmp = System.getProperty(key);
        if (tmp != null) {
            return tmp;
        }

        tmp = System.getenv(key);
        return tmp;
    }
}

File: src/main/java/no/jstien/bikeapi/utils/VarUtils.kt
package no.jstien.bikeapi.utils

object VarUtils {
    fun getEnv(key: String): String? {
        var tmp: String? = null
        tmp = System.getProperty(key)
        if (tmp != null) {
            return tmp
        }

        tmp = System.getenv(key)
        return tmp
    }
}
